HumanControls/JoystickAxis.cpp

```diff
@@ -28,37 +22,32 @@ void JoystickAxis::update()
 
 double JoystickAxis::calcResult(int rawValue)
 {
-    double value = rawValue;
+    int value = rawValue;
```

Comment:
this variable isn't necessary. Just use the `rawValue` parameter.

Reply:
Done